src/index.js

```diff
@@ -1,22 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { BrowserRouter } from 'react-router-dom';
 import './index.css';
-import { Provider } from 'react-redux';
 // eslint-disable-next-line
 import App from './App';
-import reportWebVitals from './reportWebVitals';
-import store from './redux/configureStore';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <BrowserRouter>
-      <Provider store={store}>
-        <App />
-      </Provider>
-    </BrowserRouter>
+    <App />
   </React.StrictMode>,
 );
```

Comment:
You did a great job adding tests to the login and registration pages, however, there are some issues you have to fix.

- when I start the server I get this error, kindly fix it.
![Screenshot 2023-04-10 at 10 12 20 AM](https://user-images.githubusercontent.com/105528809/230872501-16cd7fa3-f5fb-40d1-aa19-79fcb606eac8.png)